scss/navbar.scss

```diff
@@ -0,0 +1,23 @@
+.navbar {
+  align-items: center;
+  background-color: white;
```

Comment:
this background is not needed (ignoring the fact that colors should be declared as variables)